nilearn/_utils/data_gen.py

```diff
@@ -811,50 +814,47 @@ def create_fake_bids_dataset(base_dir='',
 
     n_sub : :obj:`int`, optional
         Number of subjects to be simulated in the dataset.
-        Default=10.
 
     n_ses : :obj:`int`, optional
         Number of sessions to be simulated in the dataset.
         Ignored if no_session=True.
-        Default=2.
 
     tasks : :obj:`list` of :obj:`str`, optional
         List of tasks to be simulated in the dataset.
-        Default=["localizer", "main"].
 
     n_runs : :obj:`list` of :obj:`int`, optional
         Number of runs to create, where each element indicates the
         number of runs for the corresponding task.
         The length of this list must match the number of items in ``tasks``.
         Each run creates 100 volumes.
-        Default=[1, 3].
 
     with_derivatives : :obj:`bool`, optional
         In the case derivatives are included, they come with two spaces and
         descriptions. Spaces are 'MNI' and 'T1w'. Descriptions are 'preproc'
         and :term:`fMRIPrep`. Only space 'T1w' include both descriptions.
-        Default=True.
 
     with_confounds : :obj:`bool`, optional
         Whether to generate associated confounds files or not.
-        Default=True.
 
     confounds_tag : :obj:`str` (filename suffix), optional
         If generating confounds, what path should they have? Defaults to
         `desc-confounds_timeseries` as in :term:`fMRIPrep` >= 20.2
         but can be other values (e.g. "desc-confounds_regressors" as
         in :term:`fMRIPrep` < 20.2).
-        Default="desc-confounds_timeseries".
 
     no_session : :obj:`bool`, optional
         Specifying no_sessions will only produce runs and files without the
         optional session field. In this case n_ses will be ignored.
-        Default=False.
 
     random_state : :obj:`int` or :obj:`numpy.random.RandomState` instance, \
                    optional
         Random number generator, or seed.
-        Default=0.
+
+    entities : :obj:`list`, optional
```

Comment:
In some doc strings types are mentioned as 

```
:obj:`int` 

```
and in some just as 

```
int
```

We may want to stick to only one.

Reply:
I think for the builtin classes (list, str, int, float, etc.), Sphinx will link to the class definitions without the `:obj:`, but for other classes we do need it for Sphinx to generate a link.

Reply:
ok good to know

thanks for the explanation

Reply:
@tsalo I wasn't aware that this was automatic for builtin classes. Can you link an example?

Reply:
Ah, fair enough. Maybe it's dependent on the Sphinx version, or an extension like Napoleon.

Reply:
so maybe easier then to make it explicit and systematic as a formatting style, right?

Reply:
Sounds good to me